docs/source/60_building_apps.md

```diff
@@ -187,7 +187,10 @@ object MyExample {
 ```
 
 Create an Apache Spark configuration `SparkConf` and use it to create a new `SparkContext`.
-The following serialization configuration needs to be present to register ADAM classes.
+The following serialization configuration needs to be present to register ADAM classes. This
+assumes that you do not need to register additional custom
+[Kryo Serializers](https://github.com/EsotericSoftware/kryo). If you do, you will need to
+[write your own registrator that calls the ADAM registrator](#registrator).
```

Comment:
`[Kyro Serializers]` &rarr; `[Kyro serializers]`

Would this read better as:

The following serialization configuration needs to be present to register ADAM classes. If any additional [Kyro serializers](https://github.com/EsotericSoftware/kryo) need to be registered, (...and why would they need to be?...), [create a registrator that delegates to the ADAM registrator](#registrator).